sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/cosmos/CosmosProperties.java

```diff
@@ -107,14 +110,6 @@ public void setAllowTelemetry(boolean allowTelemetry) {
         this.allowTelemetry = allowTelemetry;
     }
 
-    public ConnectionPolicy getConnectionPolicy() {
```

Comment:
How do we configure the connection policy now?

Reply:
the default connectionPolicy instance will be created in `CosmosClientBuilder`.
```java
    /**
     * Instantiates a new Cosmos client builder.
     */
    public CosmosClientBuilder() {
        //  Build default connection policy with direct default connection config
        this.connectionPolicy = new ConnectionPolicy(DirectConnectionConfig.getDefaultConfig());
        //  Some default values
        this.userAgentSuffix = "";
        this.throttlingRetryOptions = new ThrottlingRetryOptions();
    }
```